src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -943,6 +943,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
            expr As BoundExpression,
            diagnostics As DiagnosticBag
         ) As BoundExpression
+            If expr.Kind = BoundKind.ConditionalAccess AndAlso expr.Type Is Nothing Then
+                Dim conditionalAccess = DirectCast(expr, BoundConditionalAccess)
+                Dim access As BoundExpression = Me.MakeRValue(conditionalAccess.AccessExpression, diagnostics)
+
+                Dim resultType As TypeSymbol = access.Type
+
+                If Not resultType.IsErrorType() Then
+                    If resultType.IsValueType AndAlso Not resultType.IsRestrictedType Then
+                        If Not resultType.IsNullableType() Then
```

Comment:
Why not combine with enclosing condition?

Reply:
Ah, I see that it's just a copy.

---
In reply to: [188750122](https://github.com/dotnet/roslyn/pull/26903#discussion_r188750122) [](ancestors = 188750122)